pkg/crypt/crypt.go

```diff
@@ -12,13 +12,17 @@ import (
 
 //EncryptedServiceAccountKey uses github.com/ovotech/mantle to encrypt the
 // key string that's passed in
-func EncryptedServiceAccountKey(key, kmsKey string) (encKey []byte, err error) {
+func EncryptedServiceAccountKey(key, kmsKey string, base64Decode bool) (encKey []byte, err error) {
 	const singleLine = false
 	const disableValidation = true
 
 	var decodedKey []byte
```

Comment:
Good point. That's now been removed.

I toyed with the idea of removing the `EncryptedServiceAccountKey()` function altogether, seeing as it's now only a couple of constants and another function call. However I think it's better leaving it in for posterity in case anything outside the location package wants to use it.